task/rpm-ostree-oci-ta/0.2/rpm-ostree-oci-ta.yaml

```diff
@@ -234,7 +234,7 @@ spec:
         - mountPath: /var/lib/containers
           name: varlibcontainers
       script: |
-        syft oci-dir:/var/lib/containers/rhtap-final-image --output cyclonedx-json=/var/workdir/sbom-cyclonedx.json
+        syft oci-dir:/var/lib/containers/rhtap-final-image --output cyclonedx-json="/var/workdir/sbom-cyclonedx.json" --source-name "${IMAGE%:*}@$(cat "$(results.IMAGE_DIGEST.path)")"
```

Comment:
This `"${IMAGE%:*}@$(cat "$(results.IMAGE_DIGEST.path)")"` is changing the image to a digested form? It's surprising this isn't a property gathered earlier in the pipeline.
I had to go back to the bash reference manual to figure out that `%:*` is stripping the tag (I had thought `%:` was special syntax, but it's not).